src/screens/FeedScreen/Header.js

```diff
@@ -305,7 +305,7 @@ const _renderProfileNormal = ({
       icon: <TrashRed />,
       onPress: () => {
         refSheet.current.close();
-        onDeletePost();
+        onDeletePost(item);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `onDeletePost()` to `onDeletePost(item)` is a good one as it allows the deletion function to know which specific post to delete. However, please ensure that the `item` object contains all necessary information for the deletion process (like the post's ID). If not, you might need to pass additional parameters or modify the `item` object to include this information.
```diff
- onDeletePost();
+ onDeletePost(item);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->